src/lib/interceptor/search-request.interceptor.ts

```diff
@@ -261,7 +261,7 @@ export function SearchRequestInterceptor(crudOptions: CrudOptions, factoryOption
             if (crudOptions.routes?.[method]?.relations === false) {
                 return [];
             }
-            if (crudOptions.routes?.[method] && Array.isArray(crudOptions.routes?.[method]?.relations)) {
+            if (crudOptions.routes?.[method] && Array.isArray(crudOptions.routes[method].relations)) {
```

Comment:
Since the existence of `crudOptions.routes?.[method]` was confirmed earlier, optional chaining is unnecessary.